### ISSUE TITLE
Display QSO in the summary

### ISSUE THREAD

**Issue by f6fvy** [w7sst/MorseRunner#338]
Currently, the points, mults and score are displayed in the lower right summary.

![image](https://github.com/user-attachments/assets/0ccc66e3-1bf8-45cd-8cd8-e73690642f5d)

The number of QSOs (raw and verified) - on the first line - should also be displayed, to reflect the errors made during the practice session. To be more precise, we'd have to distinguish callsign errors from exchange errors, but that's probably more complicated.

**Comment by w7sst** [comment]
@f6fvy 
If I understand correctly, you are asking for 4 total lines:
_ | Raw | Verified
---|-------|---------
QSOs | 65 | 50
Pts | 123 | 114
Mult | 18 | 18
Score | 2214 | 2052

Is this what you have in mind?

**Comment by f6fvy** [comment]
Yes, that's exactly it. A more precise summary would be to distinguish between callsigns errors and exchanges errors, but displaying "QSO errors" is already useful.
